data/json/items/comestibles/drink.json

```diff
@@ -190,6 +190,28 @@
     "flags": [ "EATEN_HOT", "NUTRIENT_OVERRIDE" ],
     "fun": 6
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "coffee_substitute",
+    "name": "coffee substitute",
+    "name_plural": "coffee substitute",
+    "weight": 250,
+    "color": "brown",
+    "stim": 4,
+    "container": "bottle_plastic",
+    "comestible_type": "DRINK",
+    "symbol": "~",
+    "quench": 40,
+    "calories": 2,
+    "addiction_type": "caffeine",
+    "description": "Homemade not-coffee created from the Kentucky coffeetree, just like the Meskwaki tribe!  Doesn't actually have any caffeine, and is very bitter, but it'll pass in a pinch.",
```

Comment:
It doesn't have any caffeine, but grants caffeine addiction?

Reply:
It has an addiction potential of zero, meaning it won't cause any new addictions. Since it's a substitute, and was used as such by English settlers in times of poverty, I thought it would be interesting for it to work in sating existing addictions without actually causing any. Sort of a placebo effect, if you will. Is that okay?

Reply:
>It has an addiction potential of zero, meaning it won't cause any new addictions.

Ingame or IRL?

Reply:
I would imagine both. "Coffee" made from Kentucky coffeetrees has no caffeine in it, but it was used as a substitute when normal coffee wasn't available.

Reply:
The placebo effect seems like a good compromise to me, and realistic at first glance.